README.md

```diff
@@ -0,0 +1,42 @@
+#Chain adapter
+## Service overview 
+Chain adapter is a service that is used to fetch Iroha blocks very safely. Unlike the default Iroha blockchain listener implementation, the service is capable of publishing missing blocks. 
+
+The service is backed by RabbitMQ. All outgoing Iroha blocks are stored in the RabbitMQ exchange called `iroha` that fanouts all items to all bound queues. That implies that different services must use different queue names to avoid block stealing.
+
+Chain-adapter functionality may be used by utilizing the `ReliableIrohaChainListener` class. The class and the service itself are written using Kotling programming language entirely, but it has great interoperability with Java.
```

Comment:
Kotling?) 
I'd say: "The class and the service itself are written in Kotlin programming language that has great interoperability with Java."

Reply:
Oh, my bad. Thank you.